addons/main/script_mod.hpp

```diff
@@ -14,7 +14,11 @@
 #define REQUIRED_CBA_VERSION {3,15,7}
 
 #ifdef COMPONENT_BEAUTIFIED
-    #define COMPONENT_NAME QUOTE(ACE3 - COMPONENT_BEAUTIFIED)
+    #ifdef SUBCOMPONENT_BEAUTIFIED
+        #define COMPONENT_NAME QUOTE(ACE3 - COMPONENT_BEAUTIFIED - SUBCOMPONENT_BEAUTIFIED)
+    #else
+        #define COMPONENT_NAME QUOTE(ACE3 - COMPONENT_BEAUTIFIED)
+    #endif
 #else
     #define COMPONENT_NAME QUOTE(ACE3 - COMPONENT)
 #endif
```

Comment:
tool i use warns on nested if statements
can rewrite flat that will produce same results
```

#ifndef COMPONENT_BEAUTIFIED
    #define COMPONENT_BEAUTIFIED COMPONENT
#endif
#ifdef SUBCOMPONENT_BEAUTIFIED
    #define COMPONENT_NAME QUOTE(ACE3 - COMPONENT_BEAUTIFIED - SUBCOMPONENT_BEAUTIFIED)
#else
    #define COMPONENT_NAME QUOTE(ACE3 - COMPONENT_BEAUTIFIED)
#endif
```